staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -299,11 +337,11 @@ func (*Mounter) List() ([]MountPoint, error) {
 // will return true. When in fact /tmp/b is a mount point. If this situation
 // is of interest to you, don't use this function...
 func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
-	stat, err := os.Stat(file)
+	stat, err := statWithTimeout(file, statTimeout)
```

Comment:
this changes general implementation of `IsLikelyNotMountPoint`. Can you point me to code where it is being called in NFS teardown code path?

Reply:
Hey @gnufied!

> this changes general implementation of IsLikelyNotMountPoint

Could you please elaborate a little? If I understand correctly, for other cases that use `IsLikelyNotMountPoint`, the `os.Stat` should succeed and `statWithTimeout` shuold behave the same way as `os.Stat`. Maybe I misunderstood your point, did you have a specific way in mind?

>  I suspect it is not - there is also a PathExists code which calls os.Stat - I think that would hang too.

You're right, that's an oversight on my part, I will make the changes there as well and see if there's any other place where I should make the change.